nucypher/policy/models.py

```diff
@@ -52,31 +53,16 @@ def __init__(self, alice, hrac, expiration, ursula=None,
         self.kfrag = kfrag
         self.ursula = ursula
 
-        arrangement_delta = maya.now() - self.expiration
-        policy_duration = arrangement_delta.days
-        #
-        # super().__init__(author=self.alice, miner=ursula,
-        #                  value=self.deposit, lock_periods=lock_periods,
-        #                  arrangement_id=self.id())
-
     def __bytes__(self):
-        return bytes(self.alice.stamp) + bytes(
-            self.hrac()) + self.expiration.iso8601().encode()
-
-    def id(self):
```

Comment:
Neat :+1:

Reply:
Yeah, those are the best redlines in the PR right there IMO.  Good eye.